source/boards/sharing-boards.rst

```diff
@@ -28,3 +28,19 @@ To share a board with others:
 Anyone with the link will be able to view the board.
 
 Select **Regenerate Token** if you want to invalidate all the previously shared links. Confirm the action to regenerate the token.
+
+Import and export a board
+-------------------------
+
+Share a template
+----------------
+
+If you'd like to re-use a board, you can export it and then import the archive file in the channel of your choosing.
+
+To do this, select the **...** in the toolbar at the top of the board. Then select **Export board archive**. Download the archive file. Navigate to the channel where you'd like to add the exported board. Select the gear icon next to your profile avatar, and choose **Import archive**. The board you created will be added to this channel.
+
+The exported and imported board archives include card image attachments.
```

Comment:
I've moved this line up to the introductory sentence. If you agree with this proposal, let's remove this line.